### PR TITLE
sdk/js: change sei array conversion

### PR DESCRIPTION
**Pull request by panoel**

### CODE REVIEW

File: sdk/js/src/utils/array.ts [hunk 1]
```diff
@@ -109,6 +109,10 @@ export const tryUint8ArrayToNative = (
   } else if (chainId === CHAIN_ID_XPLA) {
     return humanAddress("xpla", a.slice(-20));
   } else if (chainId === CHAIN_ID_SEI) {
+    const h = uint8ArrayToHex(a);
+    if (!isLikely20ByteCosmwasm(h)) {
+      return humanAddress("sei", a);
```
Comment (evan-gray) [inline sdk/js/src/utils/array.ts:114]: could make these a one liner like
```
return humanAddress("sei", isLikely20ByteCosmwasm(h) ? a.slice(-20) : a);
```

**Review by evan-gray — CHANGES_REQUESTED** [review]
I think this change might be applicable for XPLA, Injective, and Wormchain too... I'm pretty sure the comment for wormchain is incorrect now 😞

File: sdk/js/src/utils/array.ts (hunk 1 above)
Comment (panoel) [reply sdk/js/src/utils/array.ts:114]: Fixed.

**Review by evan-gray — APPROVED** [review]

File: sdk/js/src/utils/array.test.ts [hunk 2]
```diff
@@ -33,10 +33,45 @@ test("wormchain address conversion", () => {
...
+
 test("injective address conversion", () => {
   const human = "inj180rl9ezc4389t72pc3vvlkxxs5d9jx60w9eeu3";
   const canonical = canonicalAddress(human);
   const lpadCanonical = zeroPad(canonical, 32);
   const native = tryUint8ArrayToNative(lpadCanonical, "injective");
   expect(native).toBe(human);
 });
+
+test("sei address conversion", () => {
```
Comment (kev1n-peters) [inline sdk/js/src/utils/array.test.ts:71]: Should we test a real sei address here?

Comment (panoel) [reply sdk/js/src/utils/array.test.ts:71]: 😅 How did that get in there???  I had an example from xLabs in there.  Fishing it back out.  Good catch!

**Review by evan-gray — APPROVED** [review]

**Review by kev1n-peters — APPROVED** [review]

**Pull request merged**